sklearn/ensemble/tests/test_stacking.py

```diff
@@ -178,14 +185,16 @@ def test_stacking_regressor_diabetes(cv, final_estimator, predict_params):
         assert len(result) == expected_result_length
 
     X_trans = reg.transform(X_test)
-    assert X_trans.shape[1] == 2
+    expected_column_count = 12 if passthrough else 2
+    assert X_trans.shape[1] == expected_column_count
```

Comment:
You can assert that the original values are the same as the passthrough values.

Same goes for the other asserts.

Reply:
Sounds good. Should that be a distinct test or should that be part of the existing `test_stacking_regressor_diabetes` and `test_stacking_classifier_iris`?

Reply:
I think this can be include here: 

```py
if passthrough:
    assert_allclose(...)
```